rules/CodingStyle/Rector/FuncCall/AddNamedArgumentsRector.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\Rector\FuncCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Expr\New_;
+use PhpParser\Node\Expr\StaticCall;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Name;
+use PHPStan\Reflection\ParameterReflectionWithPhpDocs;
+use PHPStan\Reflection\ReflectionProvider;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+use Rector\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use const PHP_VERSION_ID;
+
+final class AddNamedArgumentsRector extends AbstractRector
```

Comment:
make implements `Rector\VersionBonding\Contract\MinPhpVersionInterface` instead of manual check on refactor.